library/styleguideComponents/GlobalSearch/algoliaConnectedComponents/AlgoliaLogo/algoliaIcon.js

```diff
@@ -1,42 +0,0 @@
-                fill="url(#g)"
-            />
-        </svg>
-    );
-}
-
-AlgoliaIcon.propTypes = {
-    style: PropTypes.object,
-};
-
-AlgoliaIcon.defaultProps = {
-    style: undefined,
-};
```

Comment:
I think we should no remove this icon since we are going to use this immediately